### PR TITLE
Feature/ticket list4

### PR DESCRIPTION
**Pull request by pnoorossana**
screen shot 2018-06-22 at 6 29 34 pm

The ticket button doesn't work correctly.

### CODE REVIEW

**Comment by simpleman19** [comment]
I am working on this request

File: src/acme/ui/TicketList3.java [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+       
+     
+       
+//       combo1.setSelectedIndex(1);
+//       
+//       
+//       TableColumn comboCol = table.getColumnModel().getColumn(COMBO_COL);
+//       comboCol.setCellEditor(new DefaultCellEditor(combo1));
+       
+       JComboBox comboBox = new JComboBox();
```
Comment (jake6642) [inline src/acme/ui/TicketList3.java]: Are these options still in the combobox?

File: src/acme/ui/TicketList3.java [hunk 2]
```diff
@@ -0,0 +1,393 @@
...
+        
+      
+ 
+       addBtn.addActionListener(new ActionListener() {
+    	     public void actionPerformed(ActionEvent e) {
+    	    	goToTicketCreate();
+    	     }
+    	   });
+       
+       Courier tempCourier = new Courier();
```
Comment (jake6642) [inline src/acme/ui/TicketList3.java]: WIth the seed in place, there is no need for this code.

**Comment by simpleman19** [comment]
In the list it looks like there is a courier assigned to the ticket but when going to complete it, the courier seems to vanish

**Comment by dee-dee-raye** [comment]
Got the vanishing couriers worked out. Only thing left is print directions, because ticket.getPrintDirections is not returning anything at the moment. Also, the work is being done on TicketListUI.java.

**Comment by dee-dee-raye** [comment]
I did merge back in the changes Jacob made to dev. The tickets' paths are null.

**Comment by simpleman19** [comment]
I am still seeing issues with the couriers showing on the list and not showing on the complete ticket page but I am going to merge this and make a defect

**Pull request merged**